learning/dashboard/group9/readme.md

```diff
@@ -0,0 +1,105 @@
+# How to install and setup NodeJS and npm?
+- **step 1:**
+Open the Terminal on your Ubuntu system. You can do this by pressing the **"Ctrl+Alt+T"** keys simultaneously.
+- **step 2:**
+Update your package list by typing the following command in the Terminal:
+
+   `sudo apt-get update`
+ - **step3:**
+  Install Nodejs using the following command: 
+  
+    `sudo apt get install node js`
+  - **step4:**
+   Check if Node.js is installed by typing the following command:
+   
+    `node -v`
+    
+    This will display the version number of Node.js installed on your system.
+   - **step5:**
+Install NPM (Node Package Manager) by typing the following command:
+
+      `sudo apt-get install npm`
```

Comment:
Usually installing the NodeJS package installs npm alongwith. Please check if this is correct.

Reply:
ok